setup.cfg

```diff
@@ -26,6 +26,7 @@ classifiers =
     Topic :: Utilities
 keywords = test, unittest
 project_urls =
+    Changelog=https://docs.pytest.org/en/latest/changelog.html
```

Comment:
Perhaps `/stable` would be better? Not sure...

Reply:
As I understand, the 'latest' page always includes the entire changelog.

Reply:
The latest page on PyPI is about the last release, correct? If so the correct changelog would be `stable`: https://docs.pytest.org/en/stable/changelog.html

Reply:
Okay fair